spec/views/hq/orphans/show_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe "hq/orphans/show.html.haml", type: :view do
+  let(:orphans) { FactoryGirl.build_stubbed(:orphan_list) }
+  let(:orphan) { FactoryGirl.build_stubbed(:orphan, orphan_list: orphans) }
+
+  describe 'the orphan exists' do
+    before :each do
+      assign(:orphan, orphan)
```

Comment:
This is a minor style point: when you need to use a variable in the spec and also have it assigned to an instance variable in the view being tested, you can replace the lines
```ruby
let(:orphan) { build_stubbed :orphan }
before do
  assigns :orphan, orphan
  render
end
```
with
```ruby
before do
  @orphan = build_stubbed :orphan
  render
end
```
The `@orphan` variable is then used in both the view and the spec (you would need to change all references to `orphan` to `@orphan` in the `it` blocks).